platform-coop-toolkit/coop-library
# feat: custom pagination (resolves #188)

## Description

This PR adds a custom pagination method (`App\Controllers\App::getPagination()`) which is used in place of [`get_the_posts_pagination()`](https://developer.wordpress.org/reference/functions/get_the_posts_pagination/).

## Steps to test

1. Inspect pagination on "Browse all" page.
2. Inspect pagination on search results pages.
3. Add more than 12 favorites, then inspect pagination on favorites page.

**Expected behavior:** Markup and behaviour matches Pinecone specifications.

## Additional information

Not applicable.

## Related issues

- Resolves #188.
